src/main/java/Event.java

```diff
@@ -1,11 +1,20 @@
 class Event extends Task {
     private String at;
 
+    public Event(String description, boolean isDone, String at) {
+        super(description, isDone);
+        this.at = at;
+    }
+
     public Event(String description, String at) {
         super(description);
         this.at = at;
     }
 
+    public String formatForSave() {
```

Comment:
Maybe "formatSave" would be the same?

Reply:
Thanks for the suggestion! I couldn't think of a more concise name for this method so I had to put `formatForSave` 😅